File: PYTHON/rest.py
import requests
import json

base_url = "https://xkcd.com/info.0.json"

# To convert bytes in text
data = requests.get(base_url)

# json.loads to decode in text
data_json_obj = json.loads(data.text)

print('\n',data_json_obj["img"],'\n')

File: PHP/rest.php
<?php   
    $arrContextOptions=array(
        "ssl"=>array(
            "verify_peer"=>false,
            "verify_peer_name"=>false,
        ),
    );  
    
    $response = file_get_contents("https://xkcd.com/info.0.json", false, stream_context_create($arrContextOptions));
    
    $dataJson = json_decode($response, true);
    echo $dataJson['img'].PHP_EOL;
?>;
